content/en/database_monitoring/database_hosts.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Exploring Database Hosts
+kind: documentation
+description: Explore and dig into your database host health and configuration
+
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+On the [Databases page][1], you can assess the health and activity of your database hosts. Sort and filter the list to prioritize hosts with triggered alerts, high query volume, and other criteria.
+
+{{< img src="database_monitoring/databases-list.png" alt="The Databases page in Datadog" style="width:90%;" >}}
```

Comment:
I don't think this image adds value here, it can be confusing since the direction to click is sandwiched between two images. Optional, recommend removing.